Content.Server/GameTicking/GameRules/RuleSuspicion.cs

```diff
@@ -126,7 +125,7 @@ private void EndRound(Victory victory)
             }
 
             _gameTicker.EndRound(text);
-            _chatManager.DispatchServerAnnouncement($"Restarting in 10 seconds.");
+            _chatManager.DispatchServerAnnouncement("Restarting in 10 seconds.");
```

Comment:
But we should probably Loc those (and also maybe don't hardcode the restarttime?)